src/components/crewretrieval.tsx

```diff
@@ -186,17 +206,18 @@ class CrewRetrieval extends Component<CrewRetrievalProps, CrewRetrievalState> {
 				<Form>
 					<Form.Group inline>
 							<Form.Field
-								control={Checkbox}
-								label='Only show unowned crew'
-								checked={this.state.unownedOnly}
-								onChange={(e, { checked }) => this.setState({ unownedOnly: checked })}
+								control={Dropdown}
+								inline
```

Comment:
```suggestion
								inline
								selection
```

This should fix the styling discrepancy mentioned by @ussjohnjay.